http.go

```diff
@@ -210,6 +211,7 @@ func (impl *httpClientImpl) sendPostRequest(ctx context.Context, path, query str
 			Request:     req.Clone(context.TODO()),
 			RequestBody: data,
 		}
+		req = req.WithContext(context.WithValue(req.Context(), TraceSeriesKey{}, series))
```

Comment:
I think it is simpler to call `impl.sendRequest` directly with `series` as a parameter.

Reply:
Done